### PR TITLE
added 2 sample project in `/examples`

### PR DESCRIPTION
**Pull request by fuxingloh**
#### What kind of PR is this?:

/kind chore

#### What this PR does / why we need it:

Added 2 sample projects in `/examples`:

1. `/ocean-dex-api` - Ocean DEX API - This is a very simple example to fetch live DEX Pool Pair information from ocean.defichain.com.
1. `/ocean-masternodes-api` - DeFiChain Masternode - A very simple example to fetch Masternode information from ocean.defichain.com with pagination.

These 2 are the first of many example projects that will be added to `DeFiCh/jellyfish`. As part of the consolidation effort https://github.com/DeFiCh/jellyfish/issues/580, this is a great place to put all and everything related to the jellyfish ecosystem. Be it an example project or general API/SDK documentation. 

We will be adding more projects of different use cases and calibrate over the coming weeks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,9 @@ parcel-bundle-reports
 
 # node_modules
 node_modules
+package-lock.json
```
Comment (jingyi2811) [inline .gitignore:26]: I feel like there is something wrong here

Comment (jingyi2811) [reply .gitignore:26]: I am unsure why you wrote `package-lock.json `and` !/package-lock.json`

File: examples/ocean-masternodes-api/pages/index.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+export default function Index ({
+  masternodes,
+  nextToken
+}) {
+  const [list, setList] = useState(masternodes)
+  const [token, setToken] = useState(nextToken)
+
+  async function loadMore () {
+    if (next === undefined) {
```
Comment (jingyi2811) [inline examples/ocean-masternodes-api/pages/index.jsx:18]: Is `next` a keyword in Next js?

Comment (fuxingloh) [reply examples/ocean-masternodes-api/pages/index.jsx]: thanks! it's a mistake

File: .gitignore (hunk 1 above)
Comment (fuxingloh) [reply .gitignore:26]: to prevent other `package-lock.json` from being committed as there should only be one `package-lock.json` at the root.

**Review by jingyi2811 — APPROVED** [review]

**Pull request merged**